django_jinja/views/__init__.py

```diff
@@ -46,22 +47,24 @@ def put(self, request, *args, **kwargs):
     def patch(self, request, *args, **kwargs):
         return self.get(request, *args, **kwargs)
 
+    default_extension = get_default_extension() or ".jinja"
```

Comment:
With the latest changes, get_default_extension() is again called at compile time, and I think this should be done at runtime.

Don't worry, I'll try to take you code and adapt it.